documentation/config_file_puppetserver.markdown

```diff
@@ -26,6 +26,8 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 
     * `gem-home`: The location where JRuby looks for gems. It is also used by the `puppetserver gem` command line tool. If nothing is specified, JRuby uses the Puppet default `/opt/puppetlabs/server/data/puppetserver/jruby-gems`.
 
+    * `environment-vars:` Optional. A map of environment variables to be injected from the shell environment into the the Puppet Server process.
```

Comment:
I might say something like the following instead.  What do you think?

Optional.  A map of environment variables which are made visible to Ruby code running within JRuby, for example, via the Ruby `ENV` class.  By default, the only environment variables whose values are set into JRuby from the shell are `HOME` and `PATH`.  The default value for the `GEM_HOME` environment variable in JRuby is set from the value provided for the `jruby-puppet.gem-home` key.  Note that any variable set from the map for the `environment-vars` key will override these defaults; although it would probably be best to not override `HOME`, `PATH`, or `GEM_HOME` here since these values are already configurable.